project/AppDependencies.scala

```diff
@@ -6,27 +6,25 @@ object AppDependencies {
   lazy val libraryDependencies = compile ++ test
   lazy val dependencyOverrides = overrides
 
-  private lazy val hmrcBootstrapPlay27Version = "5.8.0"
-  private lazy val hmrcSimpleReactivemongoVersion = "8.0.0-play-27"
-  private lazy val hmrcReactiveMongoTestVersion = "5.0.0-play-27"
-  private lazy val scalaJVersion = "2.4.1"
+  private lazy val bootstrapVersion = "5.16.0"
+  private lazy val hmrcReactiveMongoTestVersion = "5.0.0-play-28"
+  private lazy val scalaJVersion = "2.4.2"
   private lazy val scalatestPlusPlayVersion = "4.0.0"
   // we need to override the akka version for now as newer versions are not compatible with reactivemongo
-  private lazy val akkaVersion = "2.6.1"
+  private lazy val akkaVersion = "2.6.14"
 
   private lazy val compile = Seq(
-    "uk.gov.hmrc" %% "bootstrap-backend-play-27" % hmrcBootstrapPlay27Version,
-    "uk.gov.hmrc" %% "simple-reactivemongo" % hmrcSimpleReactivemongoVersion,
-    "uk.gov.hmrc" %% "http-metrics" % "2.1.0-play-27",
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  %  bootstrapVersion,
+    "uk.gov.hmrc"             %% "simple-reactivemongo"       % "8.0.0-play-28",
     "com.beachape" %% "enumeratum-play" % "1.5.13"
   )
 
   private lazy val test = Seq(
     "uk.gov.hmrc" %% "reactivemongo-test" % hmrcReactiveMongoTestVersion % "test,it",
     "org.scalaj" %% "scalaj-http" % scalaJVersion % "test,it",
-    "uk.gov.hmrc" %% "bootstrap-test-play-27" % hmrcBootstrapPlay27Version % "test, it",
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"   % bootstrapVersion,
     "org.scalatestplus.play" %% "scalatestplus-play" % scalatestPlusPlayVersion % "test,it",
-    "org.mockito" %% "mockito-scala-scalatest" % "1.7.1" % "test, it",
+    "org.mockito" %% "mockito-scala-scalatest" % "1.16.46" % "test, it",
     "com.typesafe.play" %% "play-test" % PlayVersion.current % "test,it",
```

Comment:
May be worth having a play with removing some of these dependencies as https://github.com/hmrc/bootstrap-play/blob/59aae16b7ed27627ad57c6f524aa6a47c54f4107/project/LibDependencies.scala#L64 suggests that play-test and scalatestplus-play are already present. Also pegdown _should_ no longer be needed